hotness/consumers.py

```diff
@@ -291,6 +291,15 @@ def _handle_anitya_update(self, upstream, package, msg):
                         for log in six.iteritems(rh_stuff['build_logs']['logs']):
                             note = 'Build log %s.' % log
                             self.bugzilla.attach_patch(log, note, bz)
+                    # Attach rebase-helper logs for another analysis
+                    if 'logs' in rh_stuff:
+                        for log in six.iteritems(rh_stuff['logs']):
```

Comment:
Is ``rh_stuff['logs']`` a dict?

- If not, then `six.iteritems(rh_stuff['logs'])` will fail:

```python
>>> import six
>>> somelist = list(range(5))
>>> for item in six.iteritems(somelist):
...     print item
... 
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    for item in six.iteritems(somelist):
  File "/usr/lib/python2.7/site-packages/six.py", line 576, in iteritems
    return iter(d.iteritems(**kw))
AttributeError: 'list' object has no attribute 'iteritems'
```

- If ``rh_stuff['logs']`` *is* a dict, then ``log`` will be a tuple.  And if ``log`` is a tuple, I think this will fail further down when we call ``self.bugzilla.attach_patch(log, ...)`` (since that argument is supposed to just be a file name).